docs/UserGuide.md

```diff
@@ -310,12 +310,11 @@ Currently these types are only supported with the `equalTo` and `equalToField` c
 
 Users can invoke the [Faker](https://github.com/DiUS/java-faker/) custom data generators to create values.
 
-All of the types supplied on the [Faker class](https://github.com/DiUS/java-faker/blob/javafaker-1.0.1/src/main/java/com/github/javafaker/Faker.java) are accessible. Methods are invoked by entering the method signature chain. For example if we want to generate job titles, then we have to find `job.java` in the [java faker source code](https://github.com/DiUS/java-faker/tree/master/src/main/java/com/github/javafaker). From this we can see that one of the methods we can invoke on `job` is `title`. To use this in the profile we would supply the field type as `faker.job.title` as shown in the following profile.
+All of the types supplied on the [Faker class](https://github.com/DiUS/java-faker/blob/javafaker-1.0.1/src/main/java/com/github/javafaker/Faker.java) are accessible. Methods are invoked by entering the method signature chain. For example if we want to generate job titles, then we have to find `job` class in the [java faker apidocs](http://dius.github.io/java-faker/apidocs/index.html). From this we can see that one of the methods we can invoke on `job` is `title`. To use this in the profile we would supply the field type as `faker.job.title` as shown in the following profile.
```

Comment:
I would prefer this to say "Java Faker API documentation" rather than just "java faker apidocs"